Emux.GameBoy/Cartridge/BufferedExternalMemory.cs

```diff
@@ -0,0 +1,94 @@
+﻿using System;
+using System.Collections.Generic;
+using System.IO;
+using System.Text;
+
+namespace Emux.GameBoy.Cartridge {
+	public class BufferedExternalMemory : IExternalMemory {
+		private Stream memoryStream, fileStream;
```

Comment:
Please prefix private fields with an underscore to conform with the style of the rest of the project (`_memoryStream`, `_fileStream`)